core/store/store.go

```diff
@@ -251,6 +253,10 @@ func initializeORM(config *orm.Config, shutdownSignal gracefulpanic.Signal) (*or
 	if err != nil {
 		return nil, errors.Wrap(err, "initializeORM#NewORM")
 	}
+	if config.DatabaseBackupEnabled() {
+		databaseBackup := periodicbackup.NewDatabaseBackup(config.DatabaseBackupFrequency(), config.DatabaseURL(), config.RootDir(), logger.Default)
+		databaseBackup.RunBackupGracefully()
+	}
```

Comment:
not sure if it's the best place to run the backup, but it's just before migrations

Reply:
It's going to clobber with the newly migrated version the moment it runs again, making it kind of useless.

Ideally we'd have a migration version in the filename, and only overwrite files with the same version number.

Reply:
yeah true - i suppose with the timestamp it should be a bit better